tools/PI/DevHome.PI/Pages/ProcessListPage.xaml

```diff
@@ -31,6 +31,7 @@
                     x:Name="FilterDropDown" HorizontalAlignment="Stretch"
                     MinHeight="36" Height="36"
```

Comment:
Keep MinHeight, remove Height. It is causing an accessibility bug (with text scaling turned up, text will be cut off)